subsys/net/lib/aws_jobs/Kconfig

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2018 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menuconfig AWS_JOBS
+	bool "AWS Jobs library"
+
+if AWS_JOBS
+
+config CLIENT_ID_LEN
```

Comment:
Shouldn't this be `CLIENT_ID_MAX_LEN`? We don't expect that the client ID is **exactly** of this size, don't we?

Reply:
yes/no. I'll rename it.